kerl

```diff
@@ -380,7 +380,7 @@ get_otp_version()
 
 get_perl_version()
 {
-    perl -v | grep version | sed $SED_OPT -e 's/.*v5\.([0-9]+)\.[0-9].*/\1/'
+    SED_OPT=$(perl -v | grep This\ is | perl -pe 's/.*v5\.([0-9]+)\.[0-9].*/\1/')
```

Comment:
SED_OPT is been set there before, so my change is not changing that behavior. But, I agree.